SimGeneral/Debugging/python/caloParticleDebugger_cfg.py

```diff
@@ -8,7 +8,7 @@
 process = cms.Process("Demo")
 
 process.load("FWCore.MessageService.MessageLogger_cfi")
-process.load('Configuration.Geometry.GeometryExtended2026D96Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D96_cff')
```

Comment:
Do you need specific geometry here? If not, please use GeometryExtended2026Default_cff, so that you script will always be forwarded port when default geometry will be changed. The current one is D98.

Reply:
This can be fixed later.

Reply:
Thanks, i'll make sure to remember that